### PR TITLE
Fix GlobalSplitUCheckMinK not working properly when custom gsu is not set

### PR DESCRIPTION
**Pull request by KKyang**

### CODE REVIEW

**Review by aazz44ss — APPROVED** [review]

**Review by Serge45 — APPROVED** [review]

**Review by jichangjichang — CHANGES_REQUESTED** [review]
please correct formula per or discussion

File: tensilelite/Tensile/Source/lib/include/Tensile/ContractionProblemPredicates.hpp [hunk 1]
```diff
@@ -1450,15 +1450,19 @@ namespace Tensile
...
 
                 virtual bool debugEval(ContractionProblemGemm const& problem,
                                        std::ostream&                 stream) const override
                 {
                     size_t minK
-                        = (problem.getParams().gsu() > 1 ? problem.getParams().gsu() : 0) * value;
+                        = (problem.getParams().gsu() > 1 ? problem.getParams().gsu() : value[1])
+                          * value[0];
                     return debugEvalCmp(
                         problem, stream, "prob", problem.boundSize(0), ">=", "sol", minK);
```
Comment (jichangjichang) [inline tensilelite/Tensile/Source/lib/include/Tensile/ContractionProblemPredicates.hpp:1469]: please sync the formula with operator()

Comment (KKyang) [reply tensilelite/Tensile/Source/lib/include/Tensile/ContractionProblemPredicates.hpp:1469]: fixed

**Review by jichangjichang — APPROVED** [review]
lgtm

**Pull request merged**